be/src/runtime/runtime_state.cpp

```diff
@@ -176,14 +176,16 @@ void RuntimeState::init_mem_trackers(const TUniqueId& query_id, MemTracker* pare
     _instance_mem_pool = std::make_unique<MemPool>();
 }
 
-void RuntimeState::init_mem_trackers(int64_t instance_mem_limit, const std::shared_ptr<MemTracker>& query_mem_tracker) {
+void RuntimeState::init_mem_trackers(const std::shared_ptr<MemTracker>& query_mem_tracker) {
```

Comment:
Does non-pipeline engine use this method?

Reply:
No, non-pipeline uses another `RuntimeState::init_mem_trackers` in L162~L177.